packages/tocco-test-util/src/context/wrapWithContext.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const wrapWithContext = (context, contextTypes, children) => {
+  const wrapperWithContext = React.createClass({
```

Comment:
We are about to update react to 15.5.0 where createClass is deprecated.
https://github.com/facebook/react/releases